components/automate-ui/src/app/pages/job-add/job-add.component.ts

```diff
@@ -98,29 +98,27 @@ export class JobAddComponent implements OnDestroy {
     }, {
       validator: profileSelectionRequiredValidator
     });
-    const defaultStart = moment();
-    const defaultEnd = moment(defaultStart).add(1, 'days');
+    const defaultStart = moment.utc();
+    const defaultEnd = moment.utc(defaultStart).add(1, 'days');
     const scheduleGroup = this.fb.group({
       name: ['', Validators.required],
       include: false,
       start: this.fb.group({
         datetime: this.fb.group({
-          month: defaultStart.month() + 1,
+          month: defaultStart.month(),
```

Comment:
This is because previously, we were listing the months as a numeric (i.e. Jan = 1) and we were populating that initial array by counting up 12 times, starting from 0. [0, 1, .... 11].  To display properly required that we add a 1 to each month.  

Now that we are displaying the list based on an array of strings ['Jan', 'Feb',.....], we need the Array to start at 0 again, otherwise it will display "Feb" for 1 in the UI.